.npmrc

```diff
@@ -1 +1,2 @@
 package-lock=false
+auto-install-peers=true
```

Comment:
Is this change needed?

Reply:
Yeap. Current config not working on Pnpm.